pkg/check/info.go

```diff
@@ -0,0 +1,37 @@
+package check
+
+import (
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	versionLabel = "version"
+)
+
+var (
+	vCenterInfoMetric = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Name:           "vsphere_vcenter_info",
+			Help:           "Information about vSphere vCenter.",
+			StabilityLevel: metrics.ALPHA,
+		},
```

Comment:
This can't be a gauge because if we want to later on aggregate it and report it via telemetry then it will be average out when aggregated.

Reply:
I checked other metrics, build_info is gauge:
https://github.com/openshift/vsphere-problem-detector/blob/389885532a1ad284914c83e458412f395ab34378/vendor/github.com/openshift/library-go/pkg/controller/controllercmd/builder.go#L225-L233

And CVO's version is gauge too:
https://github.com/openshift/cluster-version-operator/blob/dc6f4f07d8986272b538b5d6bed9e8b6412b3ce6/pkg/cvo/metrics.go#L55-L59

Therefore I don't thin we should use Counter.

When aggregating, will we get an average or what? It's only labels + their values that are interesting to us, the metric value does not really matter.